Detections/SecurityEvent/UserAccountCreatedDeleted_10m.yaml

```diff
@@ -84,25 +84,32 @@ query: |
   | where tolong(TimeDelta) >= threshold
   | project TimeDelta, creationTime, CreateEventID, CreateActivity, Computer, TargetAccount, TargetSid, UserPrincipalName, AccountUsedToCreate, SIDofAccountUsedToCreate,
   deletionTime, DeleteEventID, DeleteActivity, AccountUsedToDelete, SIDofAccountUsedToDelete
-  | extend timestamp = creationTime, AccountCustomEntity = AccountUsedToCreate, HostCustomEntity = Computer
+  | extend timestamp = creationTime
+  | extend HostName = tostring(split(Computer, ".")[0]), DomainIndex = toint(indexof(Computer, '.'))
+  | extend HostNameDomain = iff(DomainIndex != -1, substring(Computer, DomainIndex + 1), Computer)
+  | extend AccountName = tostring(split(AccountUsedToCreate, "@")[0]), AccountUPNSuffix = tostring(split(AccountUsedToCreate, "@")[1])
 entityMappings:
   - entityType: Account
     fieldMappings:
-      - identifier: FullName
-        columnName: AccountCustomEntity
+      - identifier: Name
+        columnName: AccountName
+      - identifier: UPNSuffix
```

Comment:
AccountUsedToCreate and AccountUsedToDelete are actually built from separated entity types already. This is a legacy method. You can get what you need by the following:
| extend TargetUserName = tostring(EventData.TargetUserName), TargetDomainName = tostring(EventData.TargetDomainName)

Then entity map these to Name and NTDomain.

Also, please map another entity to Account for  AccountUsedToDelete which will be SubjectUserName and SubjectDomainName.